engine/client/voice.h

```diff
@@ -70,8 +70,8 @@ typedef struct voice_state_s
 
 	// buffers
 	byte input_buffer[MAX_RAW_SAMPLES];
-	byte output_buffer[MAX_RAW_SAMPLES];
-	byte decompress_buffer[MAX_RAW_SAMPLES];
+	byte compress_buffer[MAX_RAW_SAMPLES];
+	byte decompress_buffer[MAX_CLIENTS][MAX_RAW_SAMPLES];
```

Comment:
What's the point of having one buffer per client?

Reply:
To avoid overwriting audio data of one player with other players, in case several players are talking simultaneously